src/gui/newwizard/setupwizardwindow.cpp

```diff
@@ -2,9 +2,30 @@
 
 #include <QLabel>
 
+#include "gui/application.h"
 #include "gui/owncloudgui.h"
+#include "gui/settingsdialog.h"
+#include "theme.h"
 #include "ui_setupwizardwindow.h"
 
+using namespace std::chrono_literals;
+
+namespace {
+
+using namespace OCC;
+
+QString replaceCssColors(QString stylesheet)
+{
+    QString rv = stylesheet;
+
+    rv = stylesheet.replace(QStringLiteral("wizard.backgroundColor"), Theme::instance()->wizardHeaderBackgroundColor().name());
+    rv = stylesheet.replace(QStringLiteral("wizard.fontColor"), Theme::instance()->wizardHeaderTitleColor().name());
```

Comment:
Maybe also add "link color" -> see the contrast on the screenshot with the darkblue owncloud background: The link in brighter blue is hard to read.

Reply:
Link colors cannot be altered via stylesheets, you have to set the color inline on the `a` tag in the label's HTML source code. This is already done in the OAuth2 credentials page, just need to port that over to basic auth.